nuxt.config.ts

```diff
@@ -22,10 +22,7 @@ export default defineNuxtConfig({
         class: 'datagouv-components',
       },
       link: [
-        // Cannot use `/public/favicon.png` because the reverse proxy is calling udata-front for `/`
-        // When the migration is over we can remove this static path.
-        // :AfterMigration
-        { rel: 'shortcut icon', href: 'https://static.data.gouv.fr/_themes/gouvfr/img/favicon.png' },
+        { rel: 'shortcut icon', href: '/nuxt_images/favicon.png' },
```

Comment:
Can't we just add the acl rule? I would rather target the final URL

Reply:
This is the final URL :-) I think we'll keep the `nuxt_images` folder for the time being. When we flip the switch to route everything to cdata except the API we may rename it to just `images` but it's a quick change.